docs/organizations/managing-people.md

```diff
@@ -41,7 +41,7 @@ To add members to your organization:
 
     ![Adding members to your organization](images/organization-people-add-modal.png)
 
-1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**.
+1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails an invitation to newly added members.
```

Comment:
Plain and a bit dull, but that's all we need to tell them, really.

Here's my conclusion after some investigation: https://codacy.atlassian.net/browse/DOCS-498?focusedCommentId=53390

Reply:
Maybe we can reference that the invitation email includes a link to allow the recipients to log in to Codacy:

> Codacy emails the newly added members inviting them to log in [to Codacy].

Reply:
Sounds good. I'd avoid the second "Codacy" at the end; it should be clear enough as-is.

```suggestion
1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails the newly added members inviting them to log in.
```